src/calculateRentalCost.js

```diff
@@ -29,6 +29,25 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const currentRentalCost = days * 40;
+  const minDiscount = 20;
+  const maxDiscount = 50;
+
+  if (days < 1) {
+    return 'Error';
+  }
+
+  if (days < 3) {
+    return currentRentalCost;
+  }
+
+  if (days >= 3 && days < 7) {
+    return currentRentalCost - minDiscount;
+  }
+
+  if (days >= 7) {
+    return currentRentalCost - maxDiscount;
+  }
```

Comment:
Not necessary `if` statement, if this code is reached, it should be executed.

```suggestion
  return currentRentalCost - maxDiscount;
```